chapter_convolutional-neural-networks/conv-layer.md

```diff
@@ -48,7 +48,14 @@ from torch import nn
 ```
 
 ```{.python .input}
-#@tab mxnet, pytorch
+#@tab paddle
+from d2l import paddle as d2l
+import paddle
+from paddle import nn
+```
```

Comment:
Could you merge the two code blocks as what we did for other libraries for consistency? Other notebooks also have the same issue, could you double check? Thanks!

Reply:
I can't understand the meaning that you want to express, the pytorch code is :
```
#@tab pytorch
from d2l import torch as d2l
import torch
from torch import nn
```
the paddle code is is 

```
#@tab paddle
from d2l import paddle as d2l
import paddle
from paddle import nn
```
two blocks are different, would you want me to merge pytorch and paddle blocks ?

Reply:
<img width="590" alt="Screen Shot 2022-05-23 at 4 29 01 PM" src="https://user-images.githubusercontent.com/21952216/169920397-ffb01bec-2818-41da-bbab-2921c4938d54.png">

<img width="699" alt="Screen Shot 2022-05-23 at 4 30 54 PM" src="https://user-images.githubusercontent.com/21952216/169920465-ba0766a8-69cf-49fa-a744-c49ef45927c9.png">

You can check the difference here.

Reply:
okay, I find out the difference, the paddle code displays in two blocks because of the warning log, and could you please try the following code:

```
CUDA_VISIBLE_DEVICES=“” PYTHONWARNINGS=ignore python demo.py
```
I can't modify your compiling program， can you have a try?